### ISSUE TITLE
[Bug]: Inconsistent handling of actor's `system.details.source` data

### ISSUE THREAD

**Issue by MaxPat931** [foundryvtt/foundryvtt-premium-content#483]
### What happened?

Source data is not consistent across actors, and missing for several as well

![image](https://github.com/foundryvtt/foundryvtt-premium-content/assets/86370342/fd1a0150-860f-49fc-9bc5-b76928b3b3a5)
![image](https://github.com/foundryvtt/foundryvtt-premium-content/assets/86370342/b0408ff4-bff0-4a8c-a28d-f8f45db96504)
![image](https://github.com/foundryvtt/foundryvtt-premium-content/assets/86370342/b408dbfe-8734-40a0-aa5a-04f08de49fab)

### What package are you reporting this for?

Phandelver and Below: The Shattered Obelisk

### Version

1.0.0

**Comment by MaxPat931** [comment]
It looks like the latest update did standardize all the SRD references into Custom, but there are still inconsistencies for the PBSO entries (Book or Custom, `and` or `&`, with or without `The Shattered Obelisk`)
![image](https://github.com/foundryvtt/foundryvtt-premium-content/assets/86370342/b6684cd6-e107-4f4d-9cc9-d3f161812c52)


I adapted this from something else in #macro-polo, but seemed to work well and give all the PBSO actors consistent Source data, based on whether or not an actor of a matching `name` exists in the SRD compendium, and changes the SRD matches to have the same data as the SRD compendium, and will log out the changes, just in case.

```js
const srdMonsPack = game.packs.get("dnd5e.monsters");
const pbsoMonsPack = game.packs.get("dnd-phandelver-below.pbso-bestiary")

const matches = new Set(pbsoMonsPack.index.map(e => e.name)).intersection(new Set(srdMonsPack.index.map(e => e.name)));
for (let match of matches) {
    const targetSource = await pbsoMonsPack.getDocument(pbsoMonsPack.index.getName(match)._id);
    if (targetSource.system.details.source.book === "SRD 5.1") {
        console.log(targetSource.name, "Correct!")
        continue};
    console.log("SRD-TARGETSOURCE-OLD", targetSource.name, targetSource.system.details.source)
    await targetSource.update({ 'system.details.source.book': "SRD 5.1", 'system.details.source.custom': "", 'system.details.source.license': "CC-BY-4.0" })
    console.log("SRD-TARGETSOURCE-NEW", targetSource.name, targetSource.system.details.source)
}

const uniques = new Set(pbsoMonsPack.index.map(e => e.name)).difference(new Set(srdMonsPack.index.map(e => e.name)));
for (let unique of uniques) {
    const targetSource = await pbsoMonsPack.getDocument(pbsoMonsPack.index.getName(unique)._id);
    if (targetSource.system.details.source.book === "Phandelver and Below: The Shattered Obelisk")  {
        console.log(targetSource.name, "Correct!")
        continue};
    console.log("PBSO-TARGETSOURCE-OLD", targetSource.name, targetSource.system.details.source)
    await targetSource.update({ 'system.details.source.book': "Phandelver and Below: The Shattered Obelisk", 'system.details.source.custom': "" })
    console.log("PBSO-TARGETSOURCE-NEW", targetSource.name, targetSource.system.details.source)
}
```

**Comment by mattexdee** [comment]
Fixed as part of our [1.2.0 release](<https://github.com/foundryvtt/foundryvtt-premium-content/discussions/594>).
